Menu/Menu.js

```diff
@@ -33,3 +33,52 @@ let menuItems = [
   Step 6: add the menu component to the DOM.
   
 */
+
+
+//component [...]
+const menuButton = document.querySelector('.menu-button');
+/*new update on header*/
+const header = document.querySelector('.header')
+
+//adding the menu component to the DOM
+menuItems.forEach(add => {
+  menuButton.appendChild(menuCreator(add.arr));
+})
+
+function menuCreator(arr) {
+  const menuContainer = document.createElement('div')
+  const menuC = document.createElement('div')
+  const ulC = document.createElement('ul')
+  /*Not sure if I need to make 'li' since the forEach will do so (right?) */
+
+  //appending menu div & ul [...]
+  menuContainer.appendChild(menuC);
+  /*new update on header*/
+  header.appendChild(menuC);
+  menuC.appendChild(ulC);
+
+
+  //class names [if applicable]
+  menuC.classList.add('menu')
+  /*not sure how to add the class '.menu ul li' since the forEach creates li (at least that's my understanding)*/
+
+  //forEach that creates an <li> and appends to the <ul>
+  menuItems.forEach(list => {
+    const createItems = document.createElement('li')
+    ulC.appendChild(createItems);
+    /*not sure when I should add the class '.menu ul li' to the html, I tried 'menuulli' and 'li'*/
+    createItems.classList.add('li')
+  })
+
+  //adding a click handler to menu button [...]
+  menuButton.addEventListener('click', () => {
+    //reference to menu class [maybe?] here
+    menuRef = document.querySelector('.menu')
+    menuRef.classList.toggle('menu--open')
```

Comment:
Awesome job here! This looks great. My only recommendation would be to make menuRef a global variable. You are going to need to use it to append items to your list.